service-base/src/main/java/org/eclipse/hono/servicebase/HonoServiceBase.java

```diff
@@ -61,7 +72,7 @@ public void setConfig(final HonoConfigProperties props) {
      * @param portConfigurationTracker The future to continue or fail (in case of configuration errors).
      * @throws NullPointerException if portConfigurationTracker is null.
      */
-    public void determinePortConfigurations(Future<Void> portConfigurationTracker) {
+    protected void determinePortConfigurations(Future<Void> portConfigurationTracker) {
```

Comment:
Well, thought again this might be a point that could be desirable to customize.
But as long as we don't have this situation: agree, will change it.